src/Og.php

```diff
@@ -183,6 +182,166 @@ public static function getEntityGroups(EntityInterface $entity, array $states =
   }
 
   /**
+   * Returns all group IDs associated with the given group content entity.
+   *
+   * Do not use this to retrieve group IDs associated with a user entity. Use
```

Comment:
I think that maybe we should take the extra step an throw exception if the entity type is a user - to make it really obvious using it on a user is a bad idea :)